ods_ci/tests/Tests/500__jupyterhub/multiple-gpus.robot

```diff
@@ -31,7 +31,7 @@ Verify Number Of Available GPUs Is Correct
 Verify Two Servers Can Be Spawned
     [Documentation]    Spawns two servers requesting 1 gpu each, and checks
     ...    that both can schedule and are scheduled on different nodes.
-    [Tags]    Tier1  Sanity  Resources-2GPUS
+    [Tags]    Tier1  Sanity  Resources-2GPU    NVIDIA-GPUsS
```

Comment:
```suggestion
    [Tags]    Tier1  Sanity  Resources-2GPUS    NVIDIA-GPUs
```

Reply:
thanks for catching it. I guess that' because of the "S" in "2GPUS" I didn't see..pushed the fix in all the branches